pkg/controller/infrastructure/infraflow/aliclient/actor.go

```diff
@@ -1462,3 +1629,32 @@ func contains(elems []string, elem string) bool {
 	}
 	return false
 }
+
+func parseAlicloudVswIpv6(vswIpv6Cidr string) (string, int, error) {
+	ip, ipnet, err := net.ParseCIDR(vswIpv6Cidr)
+	if err != nil {
+		return "", -1, err
+	}
+	netMaskLen, _ := ipnet.Mask.Size()
+	if netMaskLen != 64 {
+		return "", -1, fmt.Errorf("net mask is not 64")
+	}
+	ip16 := ip.To16()
+	byte8 := ip16[7]
+	vswMask := int(byte8)
+
+	ip16[7] = 0
```

Comment:
Do I get this right that you assume a `/56` prefix here with a 8 bit subnet id? Then the subnet identifier would be byte 8. Why is it then set to `0`?

Would it not make sense to document these assumptions with some comments?

Reply:
I have added comment in code.